index.html

```diff
@@ -1,203 +1,184 @@
+     <span
+      >GLCon23'<ion-icon id="naveicon" name="aperture-outline"></ion-icon
+     ></span>
+     <ul>
+      <li id="about"><a href="./new_tab/about.html">About</a></li>
+      <li><a href="#programContainer">Program</a></li>
+      <li><a href="#register">Join</a></li>
+      <li><a href="#sponsors">Sponsor</a></li>
+     </ul>
+
+     <button class="navbutt">Campaign</button>
+    </div>
+   </nav>
+
+   <nav>
+    <div class="slidingMenu">
+     <div class="closeMenu">
+      <img src="./Images/WindowDisabled.png" alt="" />
+     </div>
+     <ul>
+      <li><a href="#homepage">Home</a></li>
+      <li><a href="./new_tab/about.html"> About</a></li>
+      <li><a href="#programContainer">Program</a></li>
+      <li><a href="#featuredSpeakers">Speakers</a></li>
+     </ul>
+    </div>
+   </nav>
+  </header>
+  <main>
+   <section>
+    <div id="homepage" class="homepage">
+     <p>"Hello World"</p>
+     <h1>GLOBAL LEADERS' <br />CONFERENCE</h1>
+     <div class="Statement">
+      You are all warmly welcome yet another beautiful event being host at the
+      four corners of the globe. An international festering of talents and
+      skills.
+     </div>
+     <div class="homedate">2018.12.08(fri)_18(wed)</div>
+     <div class="venue">@Empire of State Building,USA</div>
+    </div>
+    <section>
+     <div id="programContainer" class="programContainer">
+      <h2>MAIN PROGRAM</h2>
+      <span class="line"></span>
+      <div class="aboutContainer">
+       <ul class="Program">
+        <li id="icon"><ion-icon name="school-outline"></ion-icon></li>
+        <li class="text" id="text">Lecture</li>
+        <li id="passage">
+         Pay keen attention to whatever is being taught in the lecture room
+        </li>
+       </ul>
+       <ul class="Program">
+        <li id="icon"><ion-icon name="eye-outline"></ion-icon></li>
+        <li class="text" id="text">Exhibition</li>
+        <li id="passage">
+         Lorem ipsum dolor sit Similique dolorum gfdfg g df distinctio minus.
+        </li>
+       </ul>
+       <ul class="Program">
+        <li id="icon">
+         <ion-icon name="chatbubbles-outline"></ion-icon>
+        </li>
+        <li class="text" id="text">Forum</li>
+        <li id="passage">
+         Have time to share your thoughts and opinions friends.Feel free.
+        </li>
+       </ul>
+       <ul class="Program">
+        <li id="icon">
+         <ion-icon name="clipboard-outline"></ion-icon>
+        </li>
+        <li class="text" id="text">Workshop</li>
+        <li id="passage">
+         Try creating your own work using open source license.
+        </li>
+       </ul>
+
+       <ul class="Program">
+        <li id="icon"><ion-icon name="logo-firefox"></ion-icon></li>
+        <li class="text" id="text">Ignite</li>
+        <li id="passage">
+         Lorem ipsum dolor sit amet consectetur adipisicing elit.
+        </li>
+       </ul>
+      </div>
+      <button id="register" class="Join">

```

Comment:
- [ ] please as informed by previous reviewer, under the cards is an underlined text in the desktop version not a button like in your case.
- [optional] the cards have their heading in red. You use a yellow color. Please always respect guidelines and details. 
yours
![Screenshot (1226)_LI](https://github.com/Mike47ip/JavascriptCapstone/assets/65298048/55785155-7d08-4880-9fd8-5f72aa65798d)
original
![Screenshot (1167)](https://github.com/Mike47ip/JavascriptCapstone/assets/65298048/969928f6-5854-41c4-af05-e397c6e28a06)